samples/tab-approval/csharp/README.md

```diff
@@ -0,0 +1,139 @@
+
+-   Click on Add permissions. Please make sure to grant the admin consent for the required permissions.
+13. Navigate to **Authentication**
+    If an app hasn't been granted IT admin consent, users will have to provide consent the first time they use an app.
+    Set a redirect URI:
+    * Select **Add a platform**.
+    * Select **web**.
+    * Enter the **redirect URI** for the app in the following format: `https://{Base_Url}/Auth/end`, `https://{Base_Url}/Auth/Start`. This will be the page where a successful implicit grant flow will redirect the user.
+	
+    Enable implicit grant by checking the following boxes:  
+    ✔ ID Token  
+    ✔ Access Token  
+14.  Navigate to the **Certificates & secrets**. In the Client secrets section, click on "+ New client secret". Add a description(Name of the secret) for the secret and select “Never” for Expires. Click "Add". Once the client secret is created, copy its value, it need to be placed in the appsettings.json.
+
+15. Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+
+16. Launch Visual Studio
+   - File -> Open -> Project/Solution
+   - Navigate to folder where repository is cloned then `samples/tab-approval/csharp/TabRequestApproval.sln`
+    
+17. Run ngrok - point to port 3978
+
+    ```bash
+    # ngrok http -host-header=rewrite 3978
+    ```
+ 
+18. Setup and run the bot from Visual Studio: 
+   Modify the `appsettings.json` and fill in the following details:
+   - `MicrosoftAppId` - Generated from Step 3 (Application (client) ID)is the application app id
+   - `TenantId` - Generated from Step 3(Directory (tenant) ID) is the tenant id
+   - `MicrosoftAppPassword` - Generated from Step 14, also referred to as Client secret
+   - `{Base_URL}` - Your application's base url. E.g. https://12345.ngrok.io if you are using ngrok.
+   - Press `F5` to run the project
+	 
+19. Modify the `manifest.json` in the `/AppPackage` folder and replace the following details:
+   - `{{Microsoft-App-Id}}` with Application id generated from Step 3
+   - `{Base_URL}` - Your application's base url. E.g. https://12345.ngrok.io if you are using ngrok.
+   - `{{domain-name}}` with base Url domain. E.g. if you are using ngrok it would be `https://1234.ngrok.io` then your domain-name will be `1234.ngrok.io`.
+
+20. Zip the contents of `AppPackage` folder into a `manifest.zip`, and use the `manifest.zip` to deploy in app store or add to Teams using step 19.
+
+21. Upload the manifest.zip to Teams (in the Apps view click "Upload a custom app")
+   - Go to Microsoft Teams and then go to side panel, select Apps
+   - Choose Upload a custom App
+   - Go to your project directory, the ./AppPackage folder, select the zip folder, and choose Open.
+   - Select Add in the pop-up dialog box. Your app is uploaded to Teams.    
+
+Note: App should be installed for user's manager for getting notification.
```

Comment:
App should be installed for user's manager also to get task approval notification.

Reply:
updated